### DISCUSSION TITLE
Scan time too long

### DISCUSSION THREAD

**Discussion by cyh245689173** [spring-projects-experimental/spring-boot-migrator#878] [Q&A] [answered]
Hi，I use the spring-boot-migrator to update my project,but it scanning took too long.How long is the scanning time reasonable?

**Comment by fabapp2** [comment]
Hi @cyh245689173 
How long is "too long"? 
The time obviously depends on the size of your project. 
If you're using `spring-boot-migrator.jar` there should be some feedback from the shell?

**Comment by fabapp2** [comment, marked as answer]
It depends on the size of the project but can take minutes.
